crates/hc-utils/src/get_links_and_load_type.rs

```diff
@@ -10,7 +11,7 @@ pub fn get_links_and_load_type<R: TryFrom<Entry>>(
     tag: Option<LinkTag>,
     include_latest_updated_entry: bool,
 ) -> UtilsResult<Vec<R>> {
-    let link_info = get_links(base, tag)?;
+    let link_info = get_links(base, LinkTypeRange::Full, tag)?;
```

Comment:
You can actually do this to specify the full range, which is nice. It's hard to know from looking at it, so pass it on. There are 3 instances of this in this codebase, please change all 3 if you want to make this change.

```suggestion
    let link_info = get_links(base, .., tag)?;
```